plplot/lilac.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+from lilaclib import *
+
+maintainers = [{'github': 'migueldvb'}]
```

Comment:
Could you add an email of yours here?
Since there is no public email in your github profile, 
the build machine cannot send you the build log unless you specific one address here.

Example: https://github.com/arch4edu/arch4edu/blob/master/adms/lilac.py#L4

Reply:
Yes, I will amend the commit.